README.adoc

```diff
@@ -66,27 +64,26 @@ git clone -b master --depth 1 https://github.com/couchbaselabs/mobile-travel-sam
 
 == Couchbase Server
 
-In this lesson, you will install and launch version the 6.5.x version of Couchbase Server. If there is a later version available, you can download it as well bearing in mind that the instructions have been validated with the 6.5.x version. Note that the actual UI may vary slightly depending on the version of Couchbase server that you have installed.
+In this lesson, you will install and launch v7.1.x of Couchbase Server.
+
+Screenshots below apply to Couchbase Server 7.1.1 and there will be some variation with CBS 7.0.x. However, equivalent functionality is available in 7.0.x.
```

Comment:
Replace with "However, equivalent functionality is available in earlier versions of server. Refer to documentation for compatible versions"